lib/channels/sendMail.js

```diff
@@ -0,0 +1,45 @@
+const nodemailer = require('nodemailer');
+const { google } = require('googleapis');
+require('dotenv').config();
+
+const oAuth2Client = new google.auth.OAuth2(
+  process.env.MAIL_CLIENT_ID,
+  process.env.MAIL_CLIENT_SECRET,
+  process.env.MAIL_REDIRECT_URI,
+);
+
+oAuth2Client.setCredentials({ refresh_token: process.env.MAIL_REFRESH_TOKEN });
+
+async function sendMail(emailAddress, results) {
+  let result;
+  try {
+    const accessToken = await oAuth2Client.getAccessToken();
+    const transport = nodemailer.createTransport({
+      service: 'gmail',
+      auth: {
+        type: 'OAuth2',
+        user: process.env.MAIL_USER,
+        clientId: process.env.MAIL_CLIENT_ID,
+        clientSecret: process.env.MAIL_CLIENT_SECRET,
+        refreshToken: process.env.MAIL_REFRESH_TOKEN,
+        accessToken,
+      },
+    });
+
+    const mailOptions = {
+      from: 'Team Not Special <team.notspecial@gmail.com>',
```

Comment:
Maybe not for this first version, but wondering about either extracting this into a constant at the top of the file or even into an environment variable configured in AWS. In both cases, this would be to allow for easy editing for our theoretical user

Reply:
great call